lib/modules/datasource/artifactory/index.ts

```diff
@@ -114,6 +114,6 @@ export class ArtifactoryDatasource extends Datasource {
   }
 
   private static parseReleaseTimestamp(rawText: string): string {
-    return rawText.trim().replace(regEx(/ ?-$/), '') + 'Z';
+    return rawText.split(regEx(/\s{2,}/)).filter(e => !isNaN(Date.parse(e))) + 'Z';
```

Comment:
use luxon for date parsing

Reply:
Done. Although it should be noted that the new implementation enforces a specific datestring format. Our local Artifactory instance doesn't respond to `Accept-Language` (tested with `curl`) so this shouldn't be an issue in practice; it seems reasonable to conclude that the string is always in the same format.